community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/ListIndexesProcedure.java

```diff
@@ -46,6 +49,7 @@ protected ListIndexesProcedure(ProcedureName procedureName)
         super( procedureSignature( procedureName )
                 .out( "description", Neo4jTypes.NTString )
                 .out( "state", Neo4jTypes.NTString )
+                .out( "unique", Neo4jTypes.NTBoolean )
```

Comment:
I like that, then we can in the future extend so that the `type` can control what the return contains.